tox.ini

```diff
@@ -0,0 +1,52 @@
+[tox]
+envlist = yaml,flake8,isort,travis
+skipsdist=true
+
+[testenv]
+setenv =
+    PYWIKIBOT2_NO_USER_CONFIG = 1
+# Since pywikibot fallback to the user home directory:
+    HOME={envdir}
+deps =
+    -r{toxinidir}/requirements.txt
+commands = nosetests {posargs}
+
+[testenv:flake8]
+deps = flake8==3.7.9
+commands = flake8
+
+[flake8]
+exclude =
+    .venv,
+    .tox,
+    user-config.py,
+    user-password.py
+ignore = E501,F841
```

Comment:
Is F841 for Pywikibot? I seem to remember there being something in the config that triggered this. If so, could you add an explicit ignore for that instead?

Reply:
That is a copy pasta leftover. Removing both ignores (we are using noqa for E501 already)

Reply:
But since that re-adds some defaults I'll explicitly re-add W503 which is the only of those defaults we want to keep

Reply:
What are the other rules?